src/billing/components/PaymentInfo/PaymentDisplay.tsx

```diff
@@ -0,0 +1,21 @@
+import React, {FC} from 'react'
+import {useBilling} from 'src/billing/components/BillingPage'
+
+const PaymentDisplay: FC = () => {
+  const [{paymentMethods}] = useBilling()
+  const paymentMethod =
+    paymentMethods.find(p => p.defaultPaymentMethod) || paymentMethods[0]
```

Comment:
Is this guaranteed to have at least one element?

Reply:
Ya, since this page will only show up if the user is not a free (and has therefore submitted their billing info)